posthog/api/routing.py

```diff
@@ -324,8 +324,9 @@ def _filter_queryset_by_parents_lookups(self, queryset):
         parents_query_dict = self.parents_query_dict.copy()
 
         for source, destination in self.filter_rewrite_rules.items():
-            parents_query_dict[destination] = parents_query_dict[source]
-            del parents_query_dict[source]
+            if source in parents_query_dict:
```

Comment:
I think this _should_ throw. If it isn't found then that indicates the implementation is wrong